csharp/src/Microsoft.ML.OnnxRuntime/OnnxRuntime.cs

```diff
@@ -38,41 +38,66 @@ public enum OrtLanguageProjection
     }
 
     /// <summary>
-    /// This class intializes the process-global ONNX runtime
-    /// C# API users do not need to access this, thus kept as internal
+    /// This class initializes the process-global ONNX runtime
     /// </summary>
-    internal sealed class OnnxRuntime : SafeHandle
+    public sealed class OrtEnv : SafeHandle
```

Comment:
The class that was previously hosting the native OrtEnv handle is now public sealed OrtEnv